utility/src/main/java/org/oppia/android/util/threading/ConcurrentCollections.kt

```diff
@@ -25,7 +25,7 @@ fun <K, V> ConcurrentQueueMap<K, V>.dequeue(key: K, value: V): Boolean {
 }
 
 /** Returns the [ConcurrentLinkedQueue] corresponding to the specified key in a thread-safe way. */
-internal fun <K, V> ConcurrentQueueMap<K, V>.getQueue(key: K): ConcurrentLinkedQueue<V> {
```

Comment:
Did you get an error with the internal keyword still in place? The file seems to only be referenced in util/data, so we could probably re-introduce the keyword when we modularize that package.

Reply:
![Screenshot from 2021-03-23 00-03-43](https://user-images.githubusercontent.com/63469151/112040593-6956f480-8b6b-11eb-88d4-318a493572e6.png)
@fsharpasharp I'm getting this error after re-introducing `internal`. PTAL.

Reply:
Ack. Thanks for explaining.

Reply:
@fsharpasharp is there an issue tracking modularizing util/data? It would be nice to drop a TODO here to introduce package visibility restrictions once we can. (In general, 'internal' will probably be used less often in the Bazel world which is fine since build visibility is much more flexible).